actiontext/test/template/form_helper_test.rb

```diff
@@ -28,19 +28,39 @@ def form_with(*, **)
   test "rich text area tag" do
     message = Message.new
 
-    form_with model: message, scope: :message do |form|
-      rich_text_area_tag :content, message.content, { input: "trix_input_1" }
-    end
+    concat rich_text_area_tag :content, message.content, { input: "trix_input_1" }
 
     assert_dom_equal \
-      '<form action="/messages" accept-charset="UTF-8" method="post">' \
-        '<input type="hidden" name="content" id="trix_input_1" autocomplete="off" />' \
-        '<trix-editor input="trix_input_1" class="trix-content" data-direct-upload-url="http://test.host/rails/active_storage/direct_uploads" data-blob-url-template="http://test.host/rails/active_storage/blobs/redirect/:signed_id/:filename">' \
-        "</trix-editor>" \
-      "</form>",
+      '<input type="hidden" name="content" id="trix_input_1" autocomplete="off" />' \
+      '<trix-editor input="trix_input_1" class="trix-content" data-direct-upload-url="http://test.host/rails/active_storage/direct_uploads" data-blob-url-template="http://test.host/rails/active_storage/blobs/redirect/:signed_id/:filename">' \
+      "</trix-editor>",
       output_buffer
   end
 
+  test "#rich_text_area helper" do
+    concat rich_text_area :message, :content, { input: "trix_input_1" }
+
+    assert_dom_equal \
+      '<input type="hidden" name="message[content]" id="trix_input_1" autocomplete="off" />' \
+      '<trix-editor id="message_content" input="trix_input_1" class="trix-content" data-direct-upload-url="http://test.host/rails/active_storage/direct_uploads" data-blob-url-template="http://test.host/rails/active_storage/blobs/redirect/:signed_id/:filename">' \
+      "</trix-editor>",
+      output_buffer
+  end
+
+  test "#rich_text_area helper renders the value: argument into the hidden field" do
```

Comment:
Is there a reason to not combine this test with the `"#rich_text_area helper"` test?

Also, if we're keeping this test separate, I think the convention is to write the symbol form of option names:

```suggestion
  test "#rich_text_area helper renders the :value option into the hidden field" do
```

Reply:
@jonathanhefner I kept them separate because I wanted to explicitly test the method signature. With `rich_text_area_tag`, the value is a positional argument. With `rich_text_area`, the value is derived from `options[:value]`. That different felt significant enough to cover separately.

Reply:
I think you are saying you want to keep this test separate from `"#rich_text_area_tag helper"`, which I agree with.  But I was asking why not combine this test and `"#rich_text_area helper"`?

Or am I misunderstanding what you wrote?

Reply:
I'm sorry for being unclear. To complete the thought I state above, I think it's important to include a `#rich_text_area` test that omits the value and passes an HTML `input` option _so that there's coverage that the `Hash` isn't treated as a `value` positional argument.

Similarly, having the second test for explicit treatment of the `value` option feels similarly valuable to keep.

Does that clarify my original comment?